calico/security/encrypt-cluster-pod-traffic.md

```diff
@@ -132,22 +132,46 @@ To install WireGuard for OpenShift v4.8:
 <label:Operator,active:true>
 <%
 
-Enable WireGuard encryption across all the nodes using the following command.
+Enable IPv4 WireGuard encryption across all the nodes using the following command.
 
 ```bash
 kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"wireguardEnabled":true}}'
 ```
 
+Enable IPv6 WireGuard encryption across all the nodes using the following command.
```

Comment:
Remember that Wireguard supports v4 in v6 and v6 in v4 encap, so we should probably have a section here explaining what Calico implements when v6 / dual-stack wireguard is enabled so that it's blindingly obvious to the user what they should expect.

Reply:
Agree with Casey. Also, CE docs removed support for IPv6, so now we only have dual stack doc. https://docs.tigera.io/networking/ipv6. Was this a miss on our part to pick this to OS docs, or does OS still support IP v6?

Reply:
Calico still supports an IPv6-only mode (in addition to dual-stack)

Reply:
Good point @caseydavenport! I remember you mentioning making it clear in code comments, I'll make sure I do this in the docs as well. Thanks @caseydavenport and @bmckercher123!